src/tests/Makefile

```diff
@@ -188,6 +188,7 @@ config/eap-test: $(RADDB_PATH)mods-available/eap config/eap-test-inner-tunnel
 radiusd.pid: test.conf $(RADDB_PATH)certs/ca.pem $(RADDB_PATH)certs/server.pem
 	${Q}rm -rf $(TEST_PATH)/gdb.log $(TEST_PATH)/radius.log $(TEST_PATH)/tlscache
 	${Q}mkdir -p $(TEST_PATH)/tlscache
+	$(info ${RADIUSD_BIN} -Pxxxxml ${TEST_PATH}/radius.log -d ${top_builddir}/src/tests -n test -i 127.0.0.1 -p ${PORT} -D ${DICT_PATH})
```

Comment:
IIRC `$(info...)` prints out things when the the Makefile is loaded, not when the rule is executed.

If this is for Makefile debugging, it can be deleted.  If it's needed as log output, it should be an `echo`

Reply:
Yes, that was for me trying to understand how the build directory was structured and should be deleted.

Reply:
I will do so.